src/main/java/org/example/intro/dto/user/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,51 @@
+package org.example.intro.dto.user;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import lombok.Getter;
+import lombok.Setter;
+import org.example.intro.validation.FieldMatch;
+import org.hibernate.validator.constraints.Length;
+
+@Setter
+@Getter
+@FieldMatch(field = "password", verifyField = "repeatPassword")
+public class UserRegistrationRequestDto {
+    @NotBlank(message = "Email field is mandatory")
+    @Email(message = "Should be a valid email.")
+    @Length(max = 255, message = "Email can not be over 255 characters")
+    String email;
```

Comment:
```suggestion
    private String email;
```

all fields should be private

Reply:
++, updated. Thank you!